### ISSUE TITLE
Moving a Sense to a new Word does not round trip correctly back into FieldWorks

### ISSUE THREAD

**Issue by johnthagen** [sillsdev/TheCombine#1016]
## To Reproduce

- Create a new FW 9.1 project and add the following words

![image](https://user-images.githubusercontent.com/10340167/107807029-9676e280-6d35-11eb-98c2-59f0a521363e.png)

- File | Export | Full Lexicon. Zip the folder.
- Create a new Combine project and import Lift zip upon creation
- Data Cleanup | Merge Duplicates
- Move sense `b` into word `z`; Save
- Export Project. Extract zip file
- In Field Works | File | Import | Lift Lexion
  - Use the following options
  - ![image](https://user-images.githubusercontent.com/10340167/107807884-cb376980-6d36-11eb-8401-745253b8ff1b.png)

Note duplicate senses:

![image](https://user-images.githubusercontent.com/10340167/107807998-f326cd00-6d36-11eb-9b12-23c02b82eb55.png)

**Comment by johnthagen** [comment]
It appears the problem is that the Combine does not preserve `Sense` IDs.

Lift exported:

```xml
    <entry dateCreated="2021-02-12T18:20:03Z" dateModified="2021-02-12T18:20:12Z" id="a_832e66dc-8cc2-4efc-b55c-3b3589b027b2" guid="832e66dc-8cc2-4efc-b55c-3b3589b027b2">
        <lexical-unit>
            <form lang="fr">
                <text>a</text>
            </form>
        </lexical-unit>
        <trait name="morph-type" value="stem"/>
        <sense id="d43bff40-7873-4f9f-869f-6dbfcccdc829" order="0">
            <gloss lang="en">
                <text>b</text>
            </gloss>
        </sense>
        <sense id="cfab14c8-b68c-4e65-a32f-a3214b4b874b" order="1">
            <gloss lang="en">
                <text>c</text>
            </gloss>
        </sense>
    </entry>
    <entry dateCreated="2021-02-12T18:20:23Z" dateModified="2021-02-12T18:20:23Z" id="z_7edcd860-68a1-4d68-a5ec-d119052c5d3d" guid="7edcd860-68a1-4d68-a5ec-d119052c5d3d">
        <lexical-unit>
            <form lang="fr">
                <text>z</text>
            </form>
        </lexical-unit>
        <trait name="morph-type" value="stem"/>
        <sense id="64c38713-1caa-48d9-a22a-dbd5f03597e2">
            <gloss lang="en">
                <text>y</text>
            </gloss>
        </sense>
    </entry>
```

Combine exported:

```xml
	<entry id="a" dateCreated="2021-02-12T13:20:03Z" dateModified="2021-02-12T18:31:26Z" guid="832e66dc-8cc2-4efc-b55c-3b3589b027b2">
		<lexical-unit>
			<form lang="fr">
				<text>a</text>
			</form>
		</lexical-unit>
		<sense id="b6518833-0121-40d0-9418-49ea3cd64c69">
			<gloss lang="en">
				<text>c</text>
			</gloss>
		</sense>
	</entry>
	<entry id="z" dateCreated="2021-02-12T13:20:23Z" dateModified="2021-02-12T18:31:26Z" guid="7edcd860-68a1-4d68-a5ec-d119052c5d3d">
		<lexical-unit>
			<form lang="fr">
				<text>z</text>
			</form>
		</lexical-unit>
		<sense id="7182ae26-d7c3-4f61-a665-fbc273f3da76">
			<gloss lang="en">
				<text>y</text>
			</gloss>
		</sense>
		<sense id="271c3b0b-9dff-4bec-8a6b-63eb3142550c">
			<gloss lang="en">
				<text>b</text>
			</gloss>
		</sense>
	</entry>
```

**Comment by johnthagen** [comment]
I've confirmed that Combine preserves the `Sense` GUIDs properly on import. If you export immediately after importing, they are correct.

Losing the GUIDs must happen on merge.

**Comment by johnthagen** [comment]
This seems to have been accidently fixed by #1045.

**Comment by johnthagen** [comment]
Related FieldWorks issue: https://jira.sil.org/browse/LT-20582
